bridges/snowbridge/pallets/system/src/tests.rs

```diff
@@ -631,3 +631,177 @@ fn no_genesis_build_is_uninitialized() {
 		assert!(!EthereumSystem::is_initialized(), "Ethereum initialized.");
 	});
 }
+
+#[test]
+fn register_token_with_root_yeilds_success() {
+	new_test_ext(true).execute_with(|| {
+		let origin = RuntimeOrigin::root();
+		let location = Location::new(1, []);
+		let versioned_location: Box<VersionedLocation> = Box::new(location.clone().into());
+		let asset_metadata = AssetMetadata {
+			decimals: 10,
+			name: b"Dot".to_vec().try_into().unwrap(),
+			symbol: b"DOT".to_vec().try_into().unwrap(),
+		};
+
+		assert_ok!(EthereumSystem::register_token(origin, versioned_location, asset_metadata));
+
+		let expected_token_id =
+			hex!("03b6054d0c576dd8391e34e1609cf398f68050c23009d19ce93c000922bcd852").into();
+		let expected_location = Location::new(1, [GlobalConsensus(Kusama)]);
```

Comment:
The asset metadata says `Dot` but here it's `GlobalConsensus(Kusama)`?

Reply:
Yeah, a bit confusing and fixed in https://github.com/paritytech/polkadot-sdk/pull/5546/commits/0944d8a034404372e990edf6d29b62bf7fe5a772